app/assets/javascripts/fortitude/jquery/generic/measure-scrollbar.js

```diff
@@ -2,11 +2,19 @@
   'use strict';
 
   $.measureScrollBar = function() {
-    var $body = $('body');
-    var element = document.createElement('div');
-    element.setAttribute('style', 'position: absolute; top: -9999px; width: 50px; height: 50px; overflow: scroll;');
+    var $body = $('body'),
+        element = document.createElement('div'),
+        scrollbarWidth = element.offsetWidth - element.clientWidth;
+    
+    element.setAttribute('style', 
```

Comment:
@agius since we're already using jQuery might as well turn this into a hash so its moar readable.